Dockerfile

```diff
@@ -63,7 +68,6 @@ RUN mkdir -p $APP_ROOT && \
 WORKDIR $APP_ROOT
 
 RUN echo "gem: --no-document" > ~/.gemrc && \
-    gem install bundler -v "~> 2.4.22" && \
```

Comment:
Since this is removed, you might have to change the "BUNDLED WITH" line in the Gemfile.lock